ffi/capi/bindings/dart/lib.g.dart

```diff
@@ -279,7 +285,48 @@ class _Utf16View {
 }
 
 // ignore: unused_element
-class _BoolListView{
+class _ListUtf8View {
+  final core.List<String> _strings;
+
+  // Copies
+  _ListUtf8View(this._strings);
+
+  ffi.Pointer<_SliceUtf8> allocIn(ffi.Allocator alloc) {
+    final slice = alloc<_SliceUtf8>(length);
+    for (var i = 0; i < length; i++) {
+      final codeUnits = Utf8Encoder().convert(_strings[i]);
```

Comment:
Maybe extract the `Utf8Encoder()` to a field? It is even a const constructor, so writing `const Utf8Encoder encoder` could increase performance.

Reply:
Is that actually necessary? I'd hope the compiler doesn't need to create an empty object to call the method.

Reply:
I don't know if the compiler optimizes that, better to extract it and be sure IMO.

Reply:
prefer using `const`.

Const constructors may be used to get a constant, but can also be used to instantiate a new object. Using the const keyword ensures it is a const. (And will in AOT ensure the object is dropped if the method doesn't actually use the object.)

Reply:
A `const` field or a `const` variable?